Help the Chromium build find the rustfmt binary from the rustfmt-X.YY
package, which does not place the binary in /usr/bin/.

--- a/build/rust/rust_bindgen_generator.gni
+++ b/build/rust/rust_bindgen_generator.gni
@@ -37,7 +37,7 @@ if (!use_chromium_rust_toolchain &&
 }
 
 # Debian overrides to the above
-_rustfmt_path  = "/usr/bin/rustfmt"
+_rustfmt_path  = rust_sysroot_absolute + "/bin/rustfmt"
 _libclang_path = clang_base_path + "/lib"
 
 # Template to build Rust/C bindings with bindgen.
